internal/services/nginx/nginx_configuration_resource.go

```diff
@@ -315,7 +306,10 @@ func (m ConfigurationResource) Update() sdk.ResourceFunc {
 				return fmt.Errorf("retrieving as nil for %v", *id)
 			}
 
-			upd := existing.Model
+			upd := nginxconfiguration.NginxConfigurationRequest{
+				Name: pointer.FromString(defaultConfigurationName),
+			}
+
```

Comment:
This might end up breaking the `Update` for the resource, what is the reason for this change?

Reply:
This is still an issue @puneetsarna

Reply:
@stephybun my apologies. my comments were in a pending state and I never submitted them. 🤦 

Can you please take a look at my response above.

Reply:
good point. As I did this, I realized that the rootFile is a required property in the request. I decided to keep those lines of code to retrieve the current rootFile from the service API, set it, and then move on to the logic to see if it has changed.